tests/test_plotting.py

```diff
@@ -24,10 +24,13 @@
 LR = 1.0
 
 PLOT = False
+# plot tests cover both plotting backends
```

Comment:
I am wondering whether you can use `matplotlib.isinteractive()` from PR #960 instead of having a `PLOT` variable?

Reply:
@Kevin-Chen0 Thanks for bringing up the idea. Yes, I can add the `matplotlib.isinteractive()`. Should we fully remove the PLOT variable? Then we would not have a manual switch to turn the plotting out/on anymore. Would it be an idea to add the `matplotlib.isinteractive()` to the if queries?

Reply:
@karl-richter would be great to quickly have your opinion on it.

Reply:
If we have `matplotlib.isinteractive()`, then we should remove the PLOT variables in the tests, as it is hardcoded as False by default for the test. @karl-richter thoughts?